INSTALL.md

```diff
@@ -36,14 +36,15 @@ env CC=/usr/local/bin/gcc-6 pip install angr
 ```
 
 After installing angr, you will need to fix some shared library paths for the angr native libraries.
+Activate your virtual env and execute the following lines. A script is provided in the [angr-dev](https://github.com/angr/angr-dev) repo.
```

Comment:
Why don't you provide the link to your script directly?

Reply:
Maybe the name will change? If you want i can just link to the script directly.